e2e/schema_registry/test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
+
+docker-compose up -d zookeeper
+docker-compose up -d kafka kafka
+docker-compose up -d mysql mysql
+docker-compose up -d connect connect
+docker-compose up -d schema-registry schema-registry
+
+docker-compose up -d spark-master spark-master
+docker-compose up -d spark-worker spark-worker
+docker-compose up --exit-code-from spark-submit spark-submit
```

Comment:
it will just halt here... I don't understand, will there be any events?

Reply:
its a job that stops when its finished its ok